### ISSUE TITLE
Collision Avoidance - Train Model test_error_count is wrong if has more than two class labels

### ISSUE THREAD

**Issue by HannahZhangVW** [NVIDIA-AI-IOT/jetbot#124]
test_error_count += float(torch.sum(torch.abs(labels - outputs.argmax(1))))
will only work for two class labels.
The correct way to calculate test_error_count should be:
test_error_count += (labels != outputs.argmax(1)).sum().item()

**Comment by jaybdub** [comment]
Hi HannahZhangVW,

Yes you're correct!  The current method is only accurate for binary classes.  While this works for collision avoidance, we do hope that people will modify the notebooks for other tasks.  So we should incorporate this change when we get the opportunity.  

Best,
John
